homeassistant/components/telegram_bot/__init__.py

```diff
@@ -287,7 +287,10 @@
 def _read_file_as_bytesio(file_path: str) -> io.BytesIO:
     """Read a file and return it as a BytesIO object."""
     with open(file_path, "rb") as file:
-        return io.BytesIO(file.read())
+        data = io.BytesIO(file.read())
+        data.seek(0)
```

Comment:
I'm not sure the seek is needed as a new bytesio should read from the start

Reply:
tested